### PR TITLE
:bug: fix function type

### PR DESCRIPTION
**Pull request by AFaugeras**
Sorry, I don't know why this issue didn't appear before 😕

There are some `any` left, but I don't know how to remove them, anyway it covers most cases

### CODE REVIEW

File: packages/k-ramel/typing/index.d.ts [hunk 1]
```diff
@@ -1,54 +1,65 @@
...
-    enhancer?: any
-    init?: any
-    listeners?: any
-    devtools?: any
-    trace?: boolean
-    traceLimit?: number
-    name?: string
-    drivers?: any
-  }
+    hideRedux?: boolean;
```
Comment (fabienjuif) [inline packages/k-ramel/typing/index.d.ts]: are you sure about all the ";" here?

Comment (AFaugeras) [reply packages/k-ramel/typing/index.d.ts]: eslint did this, I'm not quite sure if this is wrong or right, well according to the documentation it seems fine.

Comment (fabienjuif) [reply packages/k-ramel/typing/index.d.ts]: Can you run the build, then I merge, and you can test the master version?

Comment (AFaugeras) [reply packages/k-ramel/typing/index.d.ts]: done ;)

**Comment by fabienjuif** [comment]
Thank you @AFaugeras :)

Keep me up to date with your tests !!

**Pull request merged**